test-vectors/credentials/create.json

```diff
@@ -552,16 +552,35 @@
     {
       "description": "creates a verifiable credential as a jwt with a did:key",
       "input": {
-        "signerDidUri": "did:key:zQ3shNLt1aMWPbWRGa8VoeEbJofJ7xJe4FCPpDKxq1NZygpiy",
-        "signerPrivateJwk": {
-          "kty": "EC",
-          "d": "OeBCt3M8roz9F9Ny192TtZXf_-qPzAkwSM6ep7QdSO4",
-          "use": "sig",
-          "crv": "secp256k1",
-          "kid": "NB_qlUIr06-AikVVMFreq0lc-omQtzc6lwhhcvgO6r4",
-          "x": "DdtN8W6x_34pB_nkxR0e1tmDkNnsJeusBAEPzKWgf_Y",
-          "y": "u3W135inodLqtcEb9jNGS3JsM_uFKmkJSb8Trc9luWI",
-          "alg": "ES256K"
+        "uri": "did:key:z6MkfUhjsZUJkzioGDULpcqxXFSNs6McMJo31txYnEaqn9dY",
+        "privateKeys": [
+          {
+            "kty": "OKP",
+            "crv": "Ed25519",
+            "use": "sig",
+            "alg": "EdDSA",
+            "kid": "c4IOrQdnehPwQZ6SyNLp9J942VCXrxgWw4zUxAHQXQE",
+            "d": "txnECNDxFc0gnD303qLJ6VfWJS8DkgpbxVbOkzzp_M4",
+            "x": "DzpSEyU0w1Myn3lA_piHAI6OrFAnZuEsTwMUPCTwMc8"
+          }
+        ],
+        "document": {
+          "id": "did:key:z6MkfUhjsZUJkzioGDULpcqxXFSNs6McMJo31txYnEaqn9dY",
```

Comment:
is it too much to ask to use a `did:jwk` instead? since we deprecated `did:key`

Reply:
this test vector specifically tests for creating a VC as a jwt with a `did:key` so i'd like to keep it here :)